src/tests/clearCart.test.js

```diff
@@ -0,0 +1,64 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import cookieParser from 'cookie-parser';
+import app from '../index.js';
+import { cartServices } from '../services';
+
+chai.should();
+chai.use(chaiHttp);
+chai.use(cookieParser);
+const { expect } = chai;
+
+describe('Get /cart items ', function () {
+  const testUserLogin = {
+    email: 'testing@example.com',
+    password: 'Qwert@12345',
+  };
+
+  it('should  return 401, when not logged in', async function () {
+    const response = await chai.request(app).delete(`/cart/clear`);
+    expect(response).to.have.status(401);
+  });
+  it('should throw 404 error, when acesses wrong path', async function () {
+    const res = await chai
+      .request(app)
+      .post('/users/login')
+      .send(testUserLogin);
+    const { token } = res.body;
+    const response = await chai
+      .request(app)
+      .delete(`/ca-rt`)
+      .set({ Authorization: `Bearer ${token}` });
+    expect(response).to.have.status(404);
+  });
+
```

Comment:
I think you should reset the state of any mocked function after each test, even if the function has not failed. Otherwise, it may affect the results of the subsequent tests. For example, you can add a beforeEach hook to reset the clearCart function. `
  beforeEach(() => {
    cartServices.clearCart = clearCart;
  });`